src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -79,31 +81,46 @@
     this.parseExterns = parseExterns;
   }
 
-  public static void main(String[] args) {
+  public static void main(String[] args) throws Exception {
     if (args.length == 0) {
       System.err.println("Usage: c2dts [FILES...]");
       System.exit(1);
     }
     List<SourceFile> sources = new ArrayList<>();
+    List<SourceFile> externs = new ArrayList<>();
+    List<SourceFile> fill = sources;
+    boolean outFile = false;
     for (String arg : args) {
+      if (arg.equals("--externs")) {
+        fill = externs;
+        continue;
+      }
+      if (arg.equals("-o")) {
+        outFile = true;
+        continue;
+      }
+      if (outFile) {
+        System.setOut(new PrintStream(new FileOutputStream(new File(arg)), true, "utf-8"));
```

Comment:
ಠ_ಠ.

I find `System.setOut` confusing and potentially dangerous. Could we explicitly allow passing down a different out stream? Also, somebody will have to close that `FileOutputStream`.